doc/usage/restructuredtext/field-lists.rst

```diff
@@ -9,19 +9,60 @@ fields marked up like this::
 
     :fieldname: Field content
 
-Sphinx provides custom behavior for bibliographic fields compared to docutils.
+Field lists are :duref:`originally defined in docutils <field-lists>`
+to show information about a page (such as the document author or date of
+publication).
+
+.. note::
+
+    The *values* of field lists will be parsed as
+    strings, you cannot use Python collections such as lists or dictionaries.
+
+Sphinx treats field lists slightly differently, as explained
+below.
 
 .. _metadata:
 
 File-wide metadata
 ------------------
 
 A field list near the top of a file is normally parsed by docutils as the
-*docinfo* which is generally used to record the author, date of publication and
-other metadata.  However, in Sphinx, a field list preceding any other markup is
-moved from the *docinfo* to the Sphinx environment as document metadata and is
-not displayed in the output; a field list appearing after the document title
-will be part of the *docinfo* as normal and will be displayed in the output.
+*docinfo* and shown on the page.  However, in Sphinx, a field list preceding
+any other markup is moved from the *docinfo* to the Sphinx environment as
+document metadata, and is not displayed in the output.
+
+.. note::
+
+    A field list appearing after the document title *will* be part of the
+    *docinfo* as normal and will be displayed in the output.
```

Comment:
This is incorrect. *docinfo* (a.k.a. bibliographic fields) is placed at top of the document except invisible elements (ex. comments). Surely "appearing after the document title" is correct. That's not all.

In addition, *docinfo* is a special naming for field-lists at the top. So it should be called as mere field-list if placed middle of document.

Reply:
Thanks for your thoughts - I am a little confused and wonder if you could help clarify. The reason I assumed that "field lists at the top are not for the docinfo in Sphinx" is because of this text [from this page](https://www.sphinx-doc.org/en/master/usage/restructuredtext/field-lists.html#file-wide-metadata):

> A field list near the top of a file is normally parsed by docutils as the docinfo which is generally used to record the author, date of publication and other metadata. However, in Sphinx, a field list preceding any other markup is moved from the docinfo to the Sphinx environment...a field list appearing after the document title will be part of the docinfo as normal and will be displayed in the output.

I believe the "note" text isn't an addition by me, it is already in the docs, I just made it a note.

Reply:
I am still not sure how to resolve this one

Reply:
Unfortunately, the current documentation is wrong.

The spec of reST format describesthe bibliographic fields (a.k.a docinfo) as following:

>When a field list is the first non-comment element in a document (after the document title, if there is one), it may have its fields transformed to document bibliographic data. This bibliographic data corresponds to the front matter of a book, such as the title page and copyright page.
https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bibliographic-fields

This means a special field-list placed at the top of the document becomes a bibliographic fields (a.k.a. docinfo). And other field-lists are mere field-lists. So "a field list appearing after the document title" is not a docinfo.